port_ocean/core/ocean_types.py

```diff
@@ -30,6 +13,18 @@ class EntitySelectorDiff(TypedDict):
 RESYNC_EVENT_LISTENER = Callable[[str], LISTENER_RESULT]
 START_EVENT_LISTENER = Callable[[], Awaitable[None]]
 
+EntitySelectorDiff = namedtuple("EntitySelectorDiff", ["passed", "failed"])
```

Comment:
please use the variation that will allow you to type it

https://stackoverflow.com/questions/34269772/type-hints-in-namedtuple